packages/web-ui/src/lab/Button/Button.props.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentPropsWithoutRef } from 'react';
+import { Responsive } from '../../types';
+
+export type ButtonProps = {
+  /**
+   * Sets the button's visual variant
+   * @default solid
+   */
+  variant?: 'solid' | 'outline' | 'ghost';
+  /**
+   * Sets the button's colour scheme
+   * @default cyan
+   */
+  colorScheme?: 'cyan' | 'red' | 'green' | 'gold' | 'grey';
```

Comment:
Is there anything in place to stop me using `variant="solid"` with `colorScheme="gold"` or `colorScheme="grey"`? If not, should there be?